src/vip_client/classes/VipSession.py

```diff
@@ -831,8 +832,10 @@ def _upload_dir(self, local_path: Path, vip_path: PurePosixPath) -> list:
         failures = []
         for local_file in files_to_upload :
             nFile+=1
-            # Get the file size (if possible)
-            try: size = f"{local_file.stat().st_size/(1<<20):,.1f}MB"
+            try:
+                size = local_file.stat().st_size
+                if size == 0: raise ValueError("Empty file")
```

Comment:
add a message : `empty files are not supported on VIP` and print the empty file path + name.